### PR TITLE
Vignette

### PR DESCRIPTION
**Pull request by LouiseDck**

### CODE REVIEW

**Comment by LouiseDck** [comment]
Closes #99

File: vignettes/getting-started.Rmd [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+1. InMemoryAnnData
+
+2. HDF5AnnData
+
+The InMemoryAnnData backend allows you to construct an AnnData object in memory.
+The HDF5AnnData backend allows you to read in an AnnData object from an `.h5ad` file.
+
+Here is an example of how to read in an `.h5ad` file and access its contents.
+
+```{r setup, eval=FALSE}
```
Comment (rcannood) [inline vignettes/getting-started.Rmd:35]: Why is this not being evaluated?

````suggestion
```{r}
````

File: vignettes/getting-started.Rmd [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+# varm <- adata$varm
+# varp <- adata$varp
+# uns <- adata$uns
+```
+
+The following example details how to make an InMemoryAnnData and access its contents.
+
+```{r inmemory}
+library(anndataR)
+
```
Comment (rcannood) [inline vignettes/getting-started.Rmd:54]: Same here:

````suggestion
```

Show the AnnData object:
```{r}
adata
```

Access the slots:
```{r}
````

**Review by rcannood — CHANGES_REQUESTED** [review]
minor changes

**Comment by rcannood** [comment]
The lintr is complaining. I would remove the commented code for now

File: vignettes/getting-started.Rmd (hunk 1 above)
Comment (LouiseDck) [reply vignettes/getting-started.Rmd:35]: I'd like  to show how to read from an .h5ad file. Is there a best practice on how to do this without cluttering the package?

Comment (rcannood) [reply vignettes/getting-started.Rmd]: The way you were doing it is correct!

**Pull request merged**